ApplicationLogs/ApplicationLogs.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo" Version="2.9.4" />
+    <PackageReference Include="Neo" Version="2.10.1" />
```

Comment:
What's this?

Reply:
It is related to commit https://github.com/neo-project/neo/commit/9831f309762ca6eedd49f02ddb3cc892287d9e2f, Erik.
I think that it will come with the next release of the neo core, right?

Reply:
You should reference to a stable release or a daily build release, instead of a non-exist one.

Reply:
@jsolman, maybe update to `2.10.1-CI00001`.
@erikzhang, are these latest daily builds manually built or automatic to the master of the Neo core?

Reply:
They are automatically built when new commit is pushed into master.

Reply:
There is no new commit yet in master that this needs...

Reply:
I got it, Erik! Do you think that we could update its name with the fix letters of the hash of the commit? :D If makes sense...Maybe the number as it is is enough.

For example:
2.10.1-CI00000 to 2.10.1-CId3094c2
2.10.1-CI00001 to 2.10.1-CI9831f30

Reply:
These both daily build needs, @jsolman:

2.10.1-CI00000 to 2.10.1-CId3094c2
2.10.1-CI00001 to 2.10.1-CI9831f30

Reply:
No build exists that has `public override void OnPluginsLoaded()` so this is better than the alternative of nothing that exists yet.

Reply:
I need https://github.com/neo-project/neo/pull/633 pushed first. Then the CI will generate a build I can reference. Until then, I have no build to reference from NuGet!

Reply:
Onhhh yes, that is true, you are right.

Reply:
Then, be patient and wait for the release. Don't make pull requests before something can be referenced.

Reply:
Next time I'll make it a draft PR. When you make changes in neo core in order to change something in plugins it makes sense to make both PRs at the same time.

Reply:
Specifically I need these extra event hooks in core in order to finish neo-project/neo-plugins/pull/52

Reply:
Some situations are hard to handle, @erikzhang. I understand your point, but these nugets are not very trivial to manage...aehauheau
Anyway...I will also try to improve the guidelines in the neo Readme for some better explanation regarding these directions.

Reply:
Actually I removed `OnPluginsLoaded()` use from this, so it can use the NuGet reference now.